Classes/Controllers/PBGitHistoryController.m

```diff
@@ -58,6 +59,14 @@ @implementation PBGitHistoryController
 
 - (void)awakeFromNib
 {
+	if (_wokeUpFromNib) {
+		return;
+	}
+	
+	_wokeUpFromNib = YES;
```

Comment:
😭. Okay, I'll have to investigate this…

Reply:
I'll take a look at this as well. I am sure there is a better solution to this..